src/autofit/afshaper.c

```diff
@@ -137,8 +137,8 @@
       hb_tag_t      tags[3];
 
 
-      hb_ot_tags_from_script_and_language( script,
-                                           HB_LANGUAGE_INVALID,
+      hb(ot_tags_from_script_and_language)( script,
```

Comment:
@lemzwerg BTW, this change is a bugfix and should go into freetype already.

Reply:
That function takes a OpenType language tag, which is a `hb_tag_t`. It *doesn't* take a `hb_language_t`, which is an ISO 639 language tag.

Reply:
OK, thanks, will do.  Please note that in the next few days I will push [this Merge Request](https://gitlab.freedesktop.org/freetype/freetype/-/merge_requests/356), together with some follow-up commits (changing `HB_LANGUAGE_INVALID` to `HB_TAG_NONE` will be part of them).  These changes also bring in some HarfBuzz-related code, which unfortunately means that it directly affects this Pull Request.

Reply:
Thanks for the headsup. I'll manage with the merge.

Reply:
> @lemzwerg BTW, this change is a bugfix and should go into freetype already.

I was wrong about this. Apologies.

Reply:
:slightly_smiling_face: